sym_api_client_python/datafeed_event_service.py

```diff
@@ -317,6 +318,14 @@ def _process_full_trace(self, id):
                 log.error("Error while computing trace results for id: " + id)
                 log.exception(exc)
 
+    @staticmethod
```

Comment:
It's not needed to be a static method, right?

Reply:
It does not use the current so IDEA suggested me to make it static

Reply:
then I guess it should be 👍 sorry I often use vim for python :)